apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.html.ts

```diff
@@ -0,0 +1,46 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { QuestionsTable, importBootstrapTable } from '../../components/QuestionsTable.html';
+import { compiledScriptTag, compiledStylesheetTag } from '../../lib/assets';
+import { QuestionsPageDataAnsified } from '../../models/questions';
+
+export const QuestionsPage = ({
+  questions,
+  showAddQuestionButton,
+  resLocals,
+}: {
+  questions: QuestionsPageDataAnsified[];
+  showAddQuestionButton: boolean;
+  resLocals;
+}) => {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../pages/partials/head') %>", resLocals)}
+        ${compiledScriptTag('instructorQuestionsClient.ts')}
+        ${compiledStylesheetTag('questionsTable.css')} ${importBootstrapTable()}
```

Comment:
I won't block you on this (it's nit), but the combination of these three would be required in all cases where `QuestionsTable` is used. Could they be a function with something like `QuestionsTableHead`? Again, feel free to ignore it.

Reply:
I had the same thought.

Reply:
done, thanks